src/main/java/me/xmrvizzy/skyblocker/skyblock/itemlist/SkyblockCraftingRecipe.java

```diff
@@ -0,0 +1,60 @@
+package me.xmrvizzy.skyblocker.skyblock.itemlist;
+
+import com.google.gson.JsonObject;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SkyblockCraftingRecipe {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SkyblockCraftingRecipe.class);
+    String craftText = "";
+    final List<ItemStack> grid = new ArrayList<>(9);
+    ItemStack result;
```

Comment:
Small nitpick.
```suggestion
    private String craftText = "";
    private final List<ItemStack> grid = new ArrayList<>(9);
    private ItemStack result;
```

Reply:
Thank you for reviewing. I've tested this a few times and it worked as intended.